src/frontend/src/apis/misc.api.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import axios from 'axios';
+import { apiUrls } from '../utils/Urls';
+
+/**
+ * Fetches the version number of the app.
+ */
+export const getVersionNumber = () => {
```

Comment:
Just realized data is being pulled apart so this might be more correct to name it "get release info" or something like that. Or pull apart the data in this function